configuration/scripts/options/set_nml.dynanderson

```diff
@@ -1,3 +1,5 @@
 kdyn            = 3
 algo_nonlin     = 'anderson'
 use_mean_vrel   = .false.
+capping_method  = 'max'
+
```

Comment:
Just wanted to double check that nothing will complain about the extra line here?